ecmd-core/pyapi/makefile

```diff
@@ -116,6 +116,15 @@ ${SOURCE_OBJS} ${SWIG_OBJS}: ${OBJPATH}%.o : %.C ${INCLUDES}
 ${SWIG_SOURCE}: ${INCLUDES} ${SWIG_INPUT}
 	@echo Swigging $@
 	${VERBOSE}${SWIG} ${SWIGFLAGS} ${DEFINES} ${PYDEFINES} -o ${SRCPATH}/${SWIG_SOURCE} -outdir ${OUTPY} ../pyapi/ecmdClientPyapi.i
+	@echo Generating mapping of output parameters
+	${VERBOSE}{ OUTARGS=$$(grep -Eo '\Wo_\w+' ${OUTPY}/${PY_TARGET} | cut -c2- | sort | uniq); \
```

Comment:
I was going through the generated output again and seeing some weirdness with APIs like ecmdGpioReadPin().
I added a line to this makefile to output the results of the grep and it is not finding any matches from this search

```
    grep -Eo '\Wo_\w+' ${OUTPY}/${PY_TARGET}
```